docs-ref-services/latest/app-configuration-readme.md

```diff
@@ -1,19 +1,18 @@
 ---
-title: 
+title: Use App Configuration client library for JavaScript
 keywords: Azure, javascript, SDK, API, @azure/app-configuration, 
 author: maggiepint
 ms.author: magpint
 ms.date: 07/07/2020
 ms.topic: article
 ms.prod: azure
-ms.technology: azure
```

Comment:
What is the guideline around the values for the `ms.prod` attribute vs `ms.technology`? I have seen validation warnings pop up saying that both cannot be set to the same value. 

I have also seen warnings for missing `ms.description`. Don't we need that for better SEO as well?

Reply:
https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/ms-prod-technology-invalid?branch=main